### PR TITLE
chore: redistribute modules to avoid failures when running in the same folder

### PR DESCRIPTION
**Pull request by manolo**

### CODE REVIEW

**Review by Artur- — CHANGES_REQUESTED** [review]
Are dependabot and issue template accidentally removed?

**Comment by manolo** [comment]
@Artur-  you were very fast. That commit was pushed by accident, I had that disabled in my fork to avoid random PRs in my repo.

File: .github/workflows/validation.yml [hunk 1]
```diff
@@ -5,15 +5,25 @@ jobs:
...
+          uname -a
+          id
+          pwd
+          type java && java -version
+          type mvn && mvn -version
+          type node && node --version
+          type npm && npm --version
       - name: Set flow version and Generate module lists
         run: |
+          # run 3 maven process at the same time and wait for them, saves half minute
```
Comment (Artur-) [inline .github/workflows/validation.yml:26]: What does this comment refer to? Where does 3 come from?

File: .github/workflows/validation.yml [hunk 2]
```diff
@@ -84,29 +97,26 @@ jobs:
...
         'test-root-context:0',
         'test-root-context:1',
         'test-root-context:2'
         ]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```
Comment (Artur-) [inline .github/workflows/validation.yml:111]: Why node 14 here and 16 above? Shouldn't it be 16 everywhere

**Review by Artur- — CHANGES_REQUESTED** [review]

File: .github/workflows/validation.yml (hunk 1 above)
Comment (manolo) [reply .github/workflows/validation.yml:26]: There are three `mvn ... &` running in background in the block, each one has a line `PN=$!` at the end there is a `wait $P1 $P2 $P3`.
It means that when setting the version we can compute modules, and save some seconds

File: .github/workflows/validation.yml (hunk 2 above)
Comment (manolo) [reply .github/workflows/validation.yml:111]: yep, hard to see why frontend compilation failed, had to play with different versions, at the end, node version was not the cause

**Review by Artur- — APPROVED** [review]

**Pull request merged**